File: ProgressNotification/app/src/main/java/com/akexorcist/progressnotification/data/ApplyPremiumAccountException.java
package com.akexorcist.progressnotification.data;

/**
 * Created by Akexorcist on 8/25/2017 AD.
 */

public class ApplyPremiumAccountException extends NullPointerException {
    private String userId;

    public ApplyPremiumAccountException(String userId) {
        this.userId = userId;
    }

    public ApplyPremiumAccountException(String s, String userId) {
        super(s);
        this.userId = userId;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }
}

File: app/src/main/java/com/akexorcist/progressnotification/data/ApplyPremiumAccountException.kt
package com.akexorcist.progressnotification.data

/**
 * Created by Akexorcist on 8/25/2017 AD.
 */

class ApplyPremiumAccountException : NullPointerException {
    var userId: String? = null

    constructor(userId: String) {
        this.userId = userId
    }

    constructor(s: String, userId: String) : super(s) {
        this.userId = userId
    }
}
